API/Controllers/BooksController.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using VL.Shared.Interfaces;
+using VL.Shared.Model;
+
+namespace API.Controllers
+{
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
+    [Route("api/vl")]
+    [ApiController]
+    public class BooksController : ControllerBase
+    {
+        private readonly IBookService _bookService;
+
+        public BooksController(IBookService bookService)
+        {
+            _bookService = bookService;
+        }
+
+        /// <summary>
+        /// Get all books
+        /// </summary>
+        /// <returns></returns>
```

Comment:
<returns></returns> is XML doc for saying it returns something

        /// <summary>
        /// Get all Books
        /// </summary>
        /// <returns>A list of books</returns>